InvenTree/gunicorn.conf.py

```diff
@@ -8,3 +8,6 @@
 
 max_requests = 1000
 max_requests_jitter = 50
+
+# preload app so that the ready functions are only executed once
+preload_app = True
```

Comment:
This will stop runtime reloads - while upstream InvenTree is not using that due to its simple plugins I know several plugins that do. This will be breaking any plugin that does so this needs to be mentioned prominently in the changelog. @wolflu05 have you measured how much this actually improves? This will force hard restarting the application when loading custom settings.py during runtime or importing global modules instead of allowing for a soft rollover of workers.

Reply:
> have you measured how much this actually improves?

This is the whole key point. Without that change, the machine init function would run in all workers which is definitely not the expected bahavior.